soc/ambiq/apollo4x/Kconfig

```diff
@@ -11,6 +11,4 @@ config SOC_SERIES_APOLLO4X
 	select CPU_HAS_ARM_MPU
 	select HAS_SWO
 	select AMBIQ_HAL
-	select CPU_HAS_DCACHE
```

Comment:
You will get errors as these are enabled in the HAL.

Reply:
I know, as I mentioned in last comment "Disable the cache kconfig is not working". And I mark the PR to "Draft" and add "DNM" label. Please try to address https://github.com/zephyrproject-rtos/zephyr/issues/72775, https://github.com/zephyrproject-rtos/zephyr/issues/73443 if time available, or we can disable the cache in both HAL and SoC kconfig.

Reply:
If they are disabling the twister test already, is there also a need to change the cake and fig and reenable the test? I don't think that makes much sense.

Reply:
Disabling the twister test for Apollo4x is just a temporary solution to unblock others' PRs and the coming 3.7.0 release. We need the twister to check our changes for Apollo4x in our next PRs. It is reporting "apollo4p_evb" is unrecognized platform when we push changes of apollo4p_evb then checks failure.